src/common-util/src/main/java/ee/ria/xroad/common/Version.java

```diff
@@ -3,17 +3,17 @@
  * Copyright (c) 2018 Estonian Information System Authority (RIA),
  * Nordic Institute for Interoperability Solutions (NIIS), Population Register Centre (VRK)
  * Copyright (c) 2015-2017 Estonian Information System Authority (RIA), Population Register Centre (VRK)
- *
+ * <p>
```

Comment:
Please configure IDE auto formatter so that it does not convert empty lines to paragraph tags (creates unnecessary changes)